src/i18n/locales/en/translation.json

```diff
@@ -957,6 +971,7 @@
     }
   },
   "common": {
+    "allow": "Allow this website to:",
```

Comment:
should the enable-related translations be put in their own section?: 
`"common": { "enable": {"allow": "..."} }`

Reply:
resolved